tests/test_base.py

```diff
@@ -63,7 +67,7 @@ def test_get_connection_params():
     opts = dict(base_opts)
     opts["OPTIONS"] = {"verify_ssl_cert": False}
     c = DatabaseWrapper(opts).get_connection_params()
-    assert c["verify_ssl_cert"] == False
+    assert not c["verify_ssl_cert"]
```

Comment:
Nit: Just in case you want to be explicitly checking for `False`. In this case, an identity match is applicable, contrary to the typical equality match.
```suggestion
    assert c["verify_ssl_cert"] is False
```

Reply:
Suggested change done. Thank you so much for the explanations. I've learned a lot.